fusion_docs/index.mdx

```diff
@@ -1,48 +1,97 @@
+With Fusion, there is nothing to install or manage. The Fusion thin client is automatically installed using [Wave](https://docs.seqera.io/wave)'s container augmentation facilities, enabling containerized applications to read from and write to cloud storage buckets as if they were local storage.
 
 ### No shared file system required
 
 To share data among pipeline tasks, organizations often turn to shared file systems such as Amazon EFS, Amazon FSx for Lustre, or NFS.
 
-Fusion avoids the need to deploy, manage, and mount shared file systems on every cloud instance by providing the same functionality over S3 – significantly reducing cost and complexity.
+Fusion removes the need to deploy, manage, and mount shared file systems on every cloud instance by providing the same functionality directly over cloud object storage – significantly reducing cost and complexity.
 
 ### Maximize pipeline performance and efficiency
 
-Copying data to and from S3 adds latency for every task, lengthening the time containers and cloud instances are deployed. This translates into longer runtimes and significantly higher costs for pipelines with thousands of tasks.
+Copying data to and from cloud storage adds latency for every task, increasing the time containers and cloud instances are deployed. This translates into longer runtimes and significantly higher costs for pipelines with thousands of tasks.
 
 Fusion eliminates these bottlenecks and delays, reducing execution time and cloud spending and using compute instances more efficiently.
 
 ### Dramatically reduce data movement
 
-When pipelines run with S3 storage, tasks typically read data from a bucket, copy it to EBS storage for processing, and copy results back to S3.
+When pipelines run with cloud storage, tasks typically read data from a bucket, copy it to compute instance storage for processing, and copy the results back to the cloud storage bucket.
 
-The result is significant overhead for every task. Fusion enables direct file access to S3 storage, eliminating unnecessary I/O and dramatically reducing data movement and overall runtime.
+The result is significant overhead for every task. Fusion enables direct file access to cloud object storage, eliminating unnecessary I/O and dramatically reducing data movement and overall runtime.
 
 ### Seamless access to cloud object storage
 
-While some open-source projects provide a POSIX interface over S3 storage, they require developers to install and configure additional software and package it in containers or VMs.
+While some open-source projects provide a POSIX interface over cloud storage, they require developers to install and configure additional software and add packages to containers or VMs.
+
+Unlike third-party solutions, Fusion is optimized for Nextflow and handles these tasks automatically. Fusion delivers fast, seamless access to cloud object storage.
+
+## Getting started
```

Comment:
I think this should be broken into a separate page at the top of the Nav (similar to how Wave separates Overview/Get Started) with a short intro.